src/dsp_tools/commands/create/models/server_project_info.py

```diff
@@ -50,11 +69,19 @@ def get_iri(self, name: str) -> str | None:
 
 @dataclass
 class CreatedIriCollection:
-    classes: set[str] = field(default_factory=set)
-    properties: set[str] = field(default_factory=set)
+    created_classes: set[str] = field(default_factory=set)
+    created_properties: set[str] = field(default_factory=set)
+    failed_classes: set[str] = field(default_factory=set)
+    failed_properties: set[str] = field(default_factory=set)
 
     def __post_init__(self) -> None:
-        self.properties.update({f"{KNORA_API_STR}seqnum", f"{KNORA_API_STR}isPartOf"})
+        self.created_properties.update({f"{KNORA_API_STR}seqnum", f"{KNORA_API_STR}isPartOf"})
+
+    def property_failed(self, props: set[str]) -> bool:
```

Comment:
It is often helpful to use a question for functions that return bool. This function answers the question: "Has  any of these properties failed?" So I would name this `has_any_property_failed()`